src/iam.ts

```diff
@@ -45,7 +46,7 @@ export interface GetPolicyCallback {
  * @param {string} [userProject] The ID of the project which will be
  *     billed for the request.
  */
-export interface SetPolicyOptions {
+export interface SetPolicyOptions extends PreconditionOptions {
```

Comment:
I might not understand well enough here, but setIamPolicy doesn't have preconditions: https://cloud.google.com/storage/docs/json_api/v1/buckets/setIamPolicy